hummingbot/client/ui/interface_utils.py

```diff
@@ -1,17 +1,17 @@
 import asyncio
 import datetime
-
-import pandas as pd
-import psutil
-from decimal import Decimal
 from typing import (
     List,
+    Optional,
     Set,
     Tuple,
-    Optional,
 )
 
-from tabulate import tabulate
+import pandas as pd
+import psutil
+import tabulate
+from decimal import Decimal
```

Comment:
The import line for Decimal should not be in the third party libraries section

Reply:
Addressed